homeassistant/components/motion_blinds/cover.py

```diff
@@ -182,7 +182,7 @@ def __init__(self, coordinator, blind, device_class, sw_version):
         else:
             via_device = (DOMAIN, blind._gateway.mac)
             connections = {}
-            name = f"{blind.blind_type}-{blind.mac[12:]}"
+            name = f"{blind.blind_type} {blind.mac[12:]}"
```

Comment:
Instead of hard coding `f"{blind.blind_type} {blind.mac[12:]}"` everywhere, add a helper function which takes the blind as input and outputs a device name.